### ISSUE TITLE
Still in development?

### ISSUE THREAD

**Issue by andrewoh531** [react-bootstrap-table/react-bootstrap-table2#94]
Hello~ 

I came here from the react-bootstrap-table thinking that version 2 was already available. It seems like it's still under development. This wasn't immediately clear to me and possibly for others. It might be worthwhile making it explicit on the homepage for anyone navigating here.

Do you have a timeline for when you expect to release the first version for react-bootstrap-table2? 

Thanks and keep up the good work!

**Comment by AllenFang** [comment]
yes.. it's still on development...  I wanna finish 80% functionality before this year and make the first release, but we need work more hardly actually. thanks

**Comment by Fatal1stZ** [comment]
is there any progress there to first release?

**Comment by AllenFang** [comment]
@Fatal1stZ first drop will be on January 2018
